frontend/features/MyPage/index.tsx

```diff
@@ -0,0 +1,143 @@
+
+  if (loading || !user || !token) return null;
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        minHeight: '100vh',
+        maxWidth: '960px',
+        width: '100%',
+        margin: '0 auto',
+      }}
+    >
+      <Box sx={{ gap: 2, padding: 2, mb: 3, width: '100%' }}>
+        <Box sx={{ width: '100%', textAlign: 'center', p: 2 }}>
+          <AccountCircleIcon sx={{ fontSize: '4rem' }} />
+        </Box>
+        <Box sx={{ width: '100%', textAlign: 'center' }}>
+          <Typography variant="h4">{user?.name}</Typography>
+        </Box>
+        <Box sx={{ width: '100%', textAlign: 'center' }}>
+          <Typography variant="h6">{user?.email}</Typography>
+        </Box>
+      </Box>
+      <Box sx={{ width: '100%', textAlign: 'center', mb: 3 }}>
+        <Button
+          sx={{
+            width: '80%',
+            backgroundColor: '#999999',
+            color: '#ffffff',
+            fontWeight: 'bold',
+            p: 2,
+            gap: 2,
+          }}
+        >
+          <CameraAlt />
+          <Typography variant="h6">雑草画像をアップロード</Typography>
+        </Button>
+        <Divider sx={{ mt: 2 }} />
+      </Box>
+      <Divider />
+      <Box sx={{ width: '80%', textAlign: 'center', mb: 3 }}>
+        <Link
+          href="/mypage/diagnoses"
+          style={{
+            display: 'inline-flex',
+            alignItems: 'center',
+            justifyContent: 'flex-start',
+            gap: '8px',
+            textDecoration: 'none',
+            color: 'inherit',
+          }}
+        >
+          <ArticleIcon sx={{ fontSize: '2rem', display: 'flex', alignItems: 'flex-start' }} />
+          <Typography variant="h6">診断結果一覧を見る</Typography>
+        </Link>
+        <Divider sx={{ mt: 2 }} />
+      </Box>
+      <Divider />
+      <Box sx={{ width: '80%', textAlign: 'center', mb: 3 }}>
+        <Link
+          href="/mypage/edit"
+          style={{
+            display: 'inline-flex',
+            alignItems: 'center',
+            justifyContent: 'flex-start',
+            gap: '8px',
+            textDecoration: 'none',
+            color: 'inherit',
+          }}
+        >
+          <EditIcon sx={{ fontSize: '2rem', display: 'flex', alignItems: 'flex-start' }} />
+          <Typography variant="h6">ユーザー情報を編集する</Typography>
+        </Link>
+        <Divider sx={{ mt: 2 }} />
+      </Box>
+      <Divider />
+      <Box sx={{ width: '80%', textAlign: 'center', mb: 3 }}>
+        <Link
+          href="/logout"
+          style={{
+            display: 'inline-flex',
+            alignItems: 'center',
+            justifyContent: 'flex-start',
+            gap: '8px',
+            textDecoration: 'none',
+            color: 'inherit',
+          }}
+        >
+          <LogoutIcon sx={{ fontSize: '2rem', display: 'flex', alignItems: 'flex-start' }} />
+          <Typography variant="h6">ログアウト</Typography>
+        </Link>
```

Comment:
```suggestion
        <Link href="/logout" style={{ textDecoration: 'none', color: 'inherit' }}>
          <Box
            sx={{
              position: 'relative',
              width: '100%',
              height: 48,
              display: 'flex',
              alignItems: 'center',
              justifyContent: 'center',
            }}
          >
            <LogoutIcon
              sx={{
                position: 'absolute',
                left: 0,
                top: '50%',
                transform: 'translateY(-50%)',
              }}
            />
            <Typography variant="h6" align="center">
              ログアウト
            </Typography>
          </Box>
        </Link>
```
一応こんな感じにしたらアイコン左寄せ、テキスト中央ってのはできそう！